src/workflows/airqo_etl_utils/ml_utils.py

```
@@ -753,6 +1046,173 @@ def process_faulty_devices_fault_sequence(df: pd.DataFrame):
+
+        predictions = classifier.predict(model_df[feature_columns])
+        probabilities = classifier.predict_proba(model_df[feature_columns])[:, 1]
+        metrics = {
+            "precision": float(
+                precision_score(model_df["weak_fault_label"], predictions)
+            ),
+            "recall": float(recall_score(model_df["weak_fault_label"], predictions)),
+            "f1_score": float(f1_score(model_df["weak_fault_label"], predictions)),
+            "positive_rate": float(model_df["weak_fault_label"].mean()),
+            "mean_probability": float(probabilities.mean()),
+            "rows_used": int(len(model_df)),
```

Ruff (0.15.2)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)